src/test/java/uk/gov/justice/hmpps/prison/repository/UserRepositoryTest.java

```diff
@@ -118,6 +118,15 @@ public void testFindUsersWithCaseloadSearch() {
         assertThat(items).extracting(UserDetail::getUsername).contains("JBRIEN");
     }
 
+    @Test
+    public void testFindUsersMulitpleRoles() {
```

Comment:
would be good to test the multiple roles code with a user that has multiple roles, difficult to tell if it works in this test since no data is returned

Reply:
I know! I was being lazy with setting up more data!  I did test on T3 locally at it works but will look at that now